packages/google-types/src/constants/mappings.ts

```diff
@@ -67,3 +68,9 @@ export const GOOGLE_TO_VOICEFLOW_SLOT_TYPE_MAP: Record<SlotType, VoiceflowConsta
   [SlotType.NUMBER]: VoiceflowConstants.SlotType.NUMBER,
   [SlotType.DATE_TIME]: VoiceflowConstants.SlotType.DATETIME,
 };
+
+export const VOICEFLOW_TO_GOOGLE_INTENT_MAP: Partial<Record<VoiceflowConstants.IntentName, GoogleIntent>> = {
+  [VoiceflowConstants.IntentName.NO]: GoogleIntent.NO,
+  [VoiceflowConstants.IntentName.YES]: GoogleIntent.YES,
+  [VoiceflowConstants.IntentName.REPEAT]: GoogleIntent.REPEAT,
```

Comment:
I believe Google also has a fallback intent, `actions.intents.fallback`. We're at least using this for DFES.

We need to create `GoogleIntent.FALLBACK`, then map a Voiceflow constant to it.

I think `VoiceflowConstants.IntentName.Fallback` should be ok?

Reply:
as mentioned [here ](https://github.com/voiceflow/general-runtime/pull/427) we cant add the the fallback intent otherwise on a DFES project an unknown choice (in choice block) makes the converstaion jump to the fallback intent instead of a NO_MATCH